### ISSUE TITLE
infrabox run should be able to use custom configured resources

### ISSUE THREAD

**Issue by vicherrero** [InfraBox/cli#30]
When running 'infrabox run', my computer has less power than the job requests. Indeed, it would be good to overwrite preconfigured resource consumption for 'infrabox run', either to increase resources to speed up or decrease them to make the execution possible.

**Comment by ib-steffen** [comment]
we could add two options `--memory` and `--cpu` to overwrite the values.

**Comment by vicherrero** [comment]
sorry, misclick

**Comment by vicherrero** [comment]
Similarly, do you think it would make sense to add an argument to pass environment variables through 'infrabox run' too? The use case is to inject environment variables in local runs that will reproduce execution in the actual cluster

**Comment by ib-steffen** [comment]
yes this makes sense as well. Should be like the docker run syntax:
https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose

@sept-en  @AndrewAhapov  please take care. thanks
